lib/analyzer.js

```diff
@@ -710,21 +710,6 @@ Analyzer.prototype.getLoadedAst = function getLoadedAst(href, loaded) {
  * @param  {Object} predicate A dom5 predicate.
  * @return {Object}
  */
-Analyzer.prototype.nodeWalkDocuments = function nodeWalkDocuments(predicate) {
-  for (var href in this.parsedDocuments) {
-    var match = dom5.nodeWalk(this.parsedDocuments[href], predicate);
-    if (match) {
-      return match;
-    }
-  }
-  return null;
-};
-
-/**
- * Calls `dom5.nodeWalkAll` on each document that `Anayzler` has laoded.
- * @param  {Object} predicate A dom5 predicate.
- * @return {Object}
- */
 Analyzer.prototype.nodeWalkAllDocuments = function nodeWalkDocuments(predicate) {
```

Comment:
The correct fix is to rename this function to `nodeWalkAllDocuments`. `nodeWalk` and `nodeWalkAll` are distinct.